src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementService.java

```diff
@@ -152,13 +156,7 @@ public ClaimDocument logUploadDocumentFailure(
         String baseFileName
     ) {
         String filename = baseFileName + ".pdf";
-
-        String errorMessage = String.format(
-            "Failure: unable to download document %s from document management due to %s",
-            filename, exception.getMessage()
-        );
-
-        logger.info(errorMessage, exception);
+        logger.warn(exception.getMessage() + " " + exception.getCause(), exception);
```

Comment:
would be better if we keep this log as this confirms retry has been done

Reply:
same as above